src/Function.h

```diff
@@ -59,6 +59,31 @@ class CVQualifiers;
 
 ///////////////////////////////////////////////////////////////////////////////
 
+class FunctionAttribute
+{
+public:
```

Comment:
Add a base class constructor and please use initializer lists: `: attribute (name)`.

Reply:
Added base class constructor `FunctionAttribute()` and using initializer list where ever required.